src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -551,6 +643,26 @@ public static uint RotateLeft(uint value, int offset)
         public static ulong RotateLeft(ulong value, int offset)
             => (value << offset) | (value >> (64 - offset));
 
+        /// <summary>
+        /// Rotates the specified value left by the specified number of bits.
+        /// Similar in behavior to the x86 instruction ROL.
+        /// </summary>
+        /// <param name="value">The value to rotate.</param>
+        /// <param name="offset">The number of bits to rotate by.
```

Comment:
Would you improve the wording of the documentation comment (I am not a native English speaker)?